lib/mirage.mli

```diff
@@ -117,6 +117,13 @@ val stdlib_random: random impl
 val nocrypto_random: random impl
 (** Passthrough to the Fortuna PRNG implemented in nocrypto. *)
 
+val default_random: random impl
+(** PRNG that will choose dynamically between {!stdlib} and {!nocrypto}.
```

Comment:
If I understand the code in `default_random` correctly, this will always be `stdlib` unless the user specifies otherwise -- this should probably be explained here, if I'm correct.